fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/services/BinaryServiceImplTest.java

```diff
@@ -133,4 +138,20 @@ public void testFindInvalidType() throws Exception {
 
         testObj.find(testSession, testPath);
     }
+
+    @Test
+    public void testFindOrCreateLocalFileBinary() throws Exception {
```

Comment:
I think this test can go away, BinaryService doesn't know anything about external content (I think it did in an earlier version). So this file can probably be reset to master.